eras/babbage/impl/src/Cardano/Ledger/Babbage/PParams.hs

```diff
@@ -239,7 +237,7 @@ instance Era era => ToCBOR (BabbagePParams Identity era) where
           !> To bppA0
           !> To bppRho
           !> To bppTau
-          !> E toCBORGroup bppProtocolVersion
+          !> To bppProtocolVersion
```

Comment:
to make sure I am following:

we want `[protocol_version]` on the wire, to be consistent with how the updates are sent on the wire. and `E toCBORGroup` inlines the two numbers. yea?

Reply:
Correct

Reply:
It inlines them, but the total number of elements in the list is not updated, thus causing it to create an invalid CBOR object, because the number of fields no longer matches the declared size. Luckily this only affects the locally stored ledger state

Reply:
great, thank you for the sanity check!